### DISCUSSION TITLE
Gardens of the Dead

### DISCUSSION THREAD

**Discussion by Groupix05** [DynamicTreesTeam/DynamicTrees#988] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/gardens-of-the-dead
1.19.2 and 1.20.1
2 new trees (1 bamboo and other its a little tree) + crimson tree

![2024-12-19_16 55 45](https://github.com/user-attachments/assets/d9fa20cb-2575-4542-843d-24d92beaf940)
![2024-12-19_16 56 24](https://github.com/user-attachments/assets/f9f36520-75ea-429c-89ba-5ccd092dd2d3)

**Comment by JarsYT** [comment]
YES

**Comment by ambientNexus** [comment]
To note, Jadens' Nether Expansion modifies the soulblight trees to have unique glowshrooms, which should probably be acknowledged for the JNE dynamic trees addon.

**Reply by Groupix05** [reply]
Yup I know that, i will add this feature once I have dt Gardens of the Dead done. Im working on this addon but I have some difficulties with custom growth, cell and feature![2025-01-04_01.10.15.png](https://github.com/user-attachments/assets/2f6affdd-8331-47ea-9428-55ebf161d0a1)

**Comment by Groupix05** [comment]
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-gardens-of-the-dead
It's in beta rn because the shroomlight feature is not working properly
